.github/ISSUE_TEMPLATE/2_new-hub.yml

```
@@ -115,56 +67,55 @@ body:
       options:
         - GitHub Authentication (e.g., @mygithubhandle)
         - Google OAuth (e.g., myemailaddress@2i2c.org)
+        - CILogon (yy
```

```suggestion
        - CILogon (yy
```

Was this an unfinished thought?

---

good catch! will make an edit